libsplinter/src/oauth/profile/github.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A profile provider that looks up GitHub profile information
+
+use reqwest::{blocking::Client, StatusCode};
+use serde::Deserialize;
+
+use crate::error::InternalError;
+
```

Comment:
Don't need a new line here; `use crate::` imports should be grouped together. This applies to the other files in this commit as well.

Reply:
Fixed